openapi/atlas-api.yaml

```diff
@@ -33902,6 +34053,7 @@ components:
         partialFilterExpression:
           type: object
           additionalProperties:
+            type: object
```

Comment:
That worries me as it is invalid use case. This field is actual array not object. 
We need to reach out to search team. For now we have transformation that fixes that problem